### ISSUE TITLE
Unable to scroll when entering reading mode

### ISSUE THREAD

**Issue by netromdk** [ZachSaucier/Just-Read#57]
On [this URL](https://www.macrumors.com/2017/03/03/iphone-8-apple-to-use-own-fingerprint-id-tech/) I cannot scroll once the reading mode is activated.

**Comment by ZachSaucier** [comment]
Hello and thanks for the report! I can't reproduce the behavior you're getting. Here are some diagnosis questions that would help me find the potential issue. I'd really appreciate if you could answer them!

What browser and operating system are you on? Has this problem occurred before on another URL? Are you using JR's auto-selection or are you manually selecting content yourself? If you retry on the page now, does the problem still occur?

**Comment by netromdk** [comment]
Hello Zach,

I haven't seen it on other sites yet. I'm running Google Chrome v. 56.0.2924.87 (64-bit) on macOS 10.12.3 (16D32).

I'm just activating the plugin with a shortcut, but it is the same behavior if I click the plugin icon to activate it. It also happens when I select only the text I want to read.

Interestingly enough, if I select "outside" the text so that it selects the entire page, then I get correct scrolling behavior.

Hope that helps.

**Comment by ZachSaucier** [comment]
I tested on a Mac and was able to recreate the error. I wasn't even able to get it working when selecting the entire page. No errors were thrown, nothing weird in the CSS was showing up, and I couldn't even force scrollbars via adding CSS in the dev tools. I honestly have no idea what is happening in this situation right now. I'm going to ask around and see if my friends can think of anything. Thanks again for the report!

**Comment by nickjenson** [comment]
Hey Zach-

I apologize for jumping-in--but I was encountering the same issue today on the same site. Inspecting the element led me to a `.simple-no-scroll"` class being added to the opening `<html>` element. 

This class had the following styles:
`.simple-no-scroll, .simple-no-scroll * {
    overflow: hidden !important;
}`

This appears to be the problem. Thanks for the extension -- I am excited to test this out further.

**Comment by ZachSaucier** [comment]
That shouldn't affect Just Read's iframe, given it's on the original document's html element, but I'll test when I have access to a Mac again.

**Comment by ZachSaucier** [comment]
Just tested on a Mac again. I can even override the element with `overflow: auto !important;` and it shows the previous (hidden) rule struck through, meaning it's *not* used any longer, but it still prevents Just Read from scrolling. It's not until I completely uncheck the hidden rule in dev tools that I am able to scroll. This must be some extreme Chrome-on-Mac-on-this-site bug, I'm pretty sure I can't do anything on my end other than remove the rule completely, which I'm not willing to do because it is necessary on pretty much every site.

**Comment by ZachSaucier** [comment]
This issue should be fixed as of JR version 1.1.14. 

However, the page still faces another issue, the same as #94. As such, I'm adding the URL to the list of problematic sites on that issue and closing this one.
